language-support/hs/bindings/src/DA/Ledger/Convert.hs

```diff
@@ -209,7 +209,7 @@ raiseCompletion :: LL.Completion -> Perhaps Completion
 raiseCompletion = \case
     LL.Completion{..} -> do
         cid <- raiseCommandId completionCommandId
-        let status = Status --TODO: stop loosing info
+        let status = completionStatus
         return Completion{..}
```

Comment:
I find this incredibly hard to read because it's not clear what variables go into `Completion{..}` and which don't. I only know that both `cid` and `status` must go into the record construction since they would be unused otherwise but I'm usually too slow (or too lazy) to run the check for unused variables in my head.
```suggestion
         return Completion{cid, status}
```
This might look like I'm contradicting myself but I don't. I can live with record wildcards in _pattern matching_ but I think they are almost always a mistake in record construction.

Reply:
I'm not really sure I understand the reasoning why one is ok and one is bad...
But I don't really care that much.
I'll change them all back again in another PR.

Reply:
I'm not a big fan of record wildcards in pattern matching but I can live with them. They implicitly bring names into scope, which is less than optimal. However, if a see a name that has not been brought into scope explicitly and there's a pattern match with a record wildcard, I can at least assume that's where the name is coming from.
If I see a record construction using a record wildcard, the only way I can figure out what actually going into that construction is to go and look at the type definition. That is something I can hardly live with.
Does that explanation make at least some sense?